packages/ckeditor5-build-multi-root/tests/ckeditor.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
In general, I believe that tests for this package are redundant and unnecessary. Manual test for "cjs version" should be enough. But I added them, as other packages do the same.